run-under-qemu.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Invocation run_under_qemu.sh [qemu_path] [test_name] [test_args...]
+
+if [ $# -lt 2 ]; then
+    echo "Usage: run_under_qemu.sh [qemu_path] [test_name] [test_args...]"
+    exit 1
+fi
+
+# Read the qemu path
+qemu_path=$1;
+
+# Read the test name
+# This will be passed to qemu as the kernel
+test_name=$2;
+
+# Remove the qemu path and test name from arguments
+# This makes $@ expand to test arguments
+shift 2
+
+if [ $# -eq 0 ]; then
+    $qemu_path -cpu I7200 -semihosting -nographic -kernel $test_name 2> /dev/null
```

Comment:
For best results, use `-m 256' and also adjust __memory_size in C/CXX flags to match this. Should reduce the number of failures you see!